### ISSUE TITLE
Improve formatting of commitsar output

### ISSUE THREAD

**Issue by tommyalatalo** [outillage/commitsar#196]
Commitsar ouputs lines in this format when commits don't follow the convention:
```
FAIL   054f4a01525dbac5dc3188d3aa6b27fd29908221   added --version flag with same output as version cmd   category missing
FAIL   21985dc84927833c8a80ad548162903f28b9ccfb   add SetLogger to db package   category missing
FAIL   304a8a71b7ed7e377dbf323e17395c4df2123232   refactored main into multiple packages, updated tests   category missing
```
The part at the end, `category missing`, is added by commitsar to point out the error, but the formatting of the lines doesn't make this very readable.

It would be better if the error was put before the commit message, in it's own column:
```
FAIL   054f4a01525dbac5dc3188d3aa6b27fd29908221   category missing   added --version flag with same output as version cmd
FAIL   21985dc84927833c8a80ad548162903f28b9ccfb   category missing   add SetLogger to db package
FAIL   304a8a71b7ed7e377dbf323e17395c4df2123232   category missing   refactored main into multiple packages, updated tests
```

This would make the output a lot more readable, just set the column width of the error message to the longest of the errors present or something like that.

**Comment by fallion** [comment]
Good use. Thanks for the feedback! It’s nice to see people using this and coming back with ideas for improvement.

**Comment by fallion** [comment]
Closed by #210
